.circleci/config.yml

```diff
@@ -17,12 +18,208 @@ jobs:
+        - run:
+            name: Setup environment
+            command: source ci/scripts/ci_setup.sh
+        - run:
+            name: Update version and code
+            command: source ci/scripts/ci_updateversion.sh
+        - run:
+            name: Create apk
+            command: ./gradlew assemble
+        - run:
+            name: Deploy to Google Play
+            command: source ci/scripts/ci_fastlane.sh "playstore"
+        - run:
+            name: Create a github release
+            command: source ci/scripts/ci_github_release.sh
+        - save_cache:
+            paths:
+              - ~/.gradle
+            key: jars-{{ checksum "build.gradle" }}-{{ checksum  "app/build.gradle" }}
+  documentation:
+      working_directory: ~/flyve_mdm
+      docker:
+        # Android the primary container
+        - image: circleci/android:api-26-alpha
+      environment:
+        JVM_OPTS: -Xmx3200m
+      steps:
+        - checkout
+        - restore_cache:
+            key: jars-{{ checksum "build.gradle" }}-{{ checksum  "app/build.gradle" }}
+        - run:
+            name: Setup environment
+            command: source ci/scripts/ci_setup.sh
+        - run:
+            name: Setup emulator
+            command: sdkmanager "system-images;android-16;default;armeabi-v7a" && echo "no" | avdmanager create avd -n test -k "system-images;android-16;default;armeabi-v7a"
+        - run:
+            name: Launch emulator
+            command: export LD_LIBRARY_PATH=${ANDROID_HOME}/emulator/lib64:${ANDROID_HOME}/emulator/lib64/qt/lib && emulator64-arm -avd test -noaudio -no-boot-anim -no-window -accel on
+            background: true
+        - run:
+            name: Wait emulator
+            command: |
+              # wait for it to have booted
+              circle-android wait-for-boot
+              # unlock the emulator screen
+              sleep 30
+              adb shell input keyevent 82
+        - run:
+            name: Coverage
+            command: source ci/scripts/ci_coverage.sh
+        - run:
+            name: Javadoc
+            command: source ci/scripts/ci_javadoc.sh
+        - save_cache:
+            paths:
+              - ~/.gradle
+            key: jars-{{ checksum "build.gradle" }}-{{ checksum  "app/build.gradle" }}
 
+workflows:
+  version: 2
+  build_test_and_deploy:
+    jobs:
+      - build
+      - test_api_version_16:
+          requires:
+            - build
+          filters:
+            branches:
+              only:
+                - develop
+      - test_api_version_25:
+          requires:
+            - build
+          filters:
+            branches:
+              only:
+                - develop
+      - deploy_beta:
+          requires:
+            - test_16
+            - test_25
+          filters:
+            branches:
+              only:
+                - develop
+      - documentation:
+          requires:
+            - test_16
```

Comment:
It's ok?
- test_16
- test_25

Or you need to change to:
- test_api_version_16
- test_api_version_25

Reply:
Ready @ajsb85 you are right, now is test_api_version_16 and test_api_version_25